templates/README.md

```diff
@@ -72,3 +72,18 @@ The `integrity` hash will automatically be filled out by the app.
   "url": "https://github.com/{OWNER}/{REPO}/archive/refs/tags/{TAG}.tar.gz"
 }
 ```
+
+---
+
+### (Optional) [config.yml](.bcr/config.yml)
```

Comment:
it's too bad to introduce the complexity of a new configuration file at the very end of the project - I assume you considered whether there's an ergonomic way to set this inside one of the existing files, like a field in metadata.template.json that only we read

Reply:
I think I prefer to keep the existing files free from anything that wouldn't be part of a BCR entry. This config file is optional and hopefully won't be needed in most cases.